rules/build_sector.smk

```diff
@@ -284,6 +284,29 @@ rule build_central_heating_temperature_profiles:
         "../scripts/build_central_heating_temperature_profiles/run.py"
 
 
+rule build_heat_source_potentials:
+    params:
+        fraunhofer_heat_sources=config_provider(
+            "sector", "district_heating", "fraunhofer_heat_utilisation_potentials"
+        ),
+        heat_source="{heat_source}",
```

Comment:
Why is that necessary? Could it not just be accessed in the script with `snakemake.wildcards.heat_source`? Just curious -- fine to leave as is if deemed necessary.

Reply:
Done - didn't know I could do that:)